core/testutils_test.go

```diff
@@ -63,3 +63,8 @@ func randOtherReplicaID(id, n uint32) uint32 {
 	offset := rand.Intn(int(n)-1) + 1
 	return (id + uint32(offset)) % n
 }
+
+// randBytes returns a slice of random bytes
+func randBytes() []byte {
+	return []byte{byte(rand.Int())} // nolint:gosec
```

Comment:
True, on one hand, this does not give any randomness between test runs, but on the other hand, makes test results more reproducible. I'm not sure what would be better. Maybe it doesn't matter much in our case because of the way we use the random values in testing.

Reply:
I don't have the answer to the question, too. I'm fine with this. Thanks.